src/application/controllers/api/v2/Appointmentsv2.php

```diff
@@ -64,26 +64,25 @@ public function get($id_integrated = null) {
         if ($id_integrated !== NULL) {
             $conditions['id_integrated'] = $id_integrated;
         }
-        
-        $appointments = $this->appointments_model_v2->get_batch($conditions, array_key_exists('aggregates', $_GET));
+        $sort = $this->input->get('sort') == null ? 'asc' : $this->input->get('sort') ;
```

Comment:
Did you check the sort function which been using in the response?
Could we use it instead?

Reply:
```
    /**
     * Supports up to 3 columns for sorting.
     *
     * Example:
     *   http://ea-installation.com/api/v1/appointments?sort=-start,+notes,-hash
     *
     * @param array &$response The response array to be processed.
```
I followed comments in Sort.php class but I'm not able to reproduce sort function supported by codeigniter. Any tips for make it works ?

Therefore, I've disabled the current sort and using the new one.